### DISCUSSION TITLE
I can't renew my Student Developer Pack

### DISCUSSION THREAD

**Discussion by imJack6** [community/community#120310] [GitHub Education]
### Hi everyone,

Last night I received an email from GitHub Education. The general content of the email was asking me to re-verify my academic status, but when I went to `https://education.github.com/benefits` (the link in the email ) and click [submit your information again](https://education.github.com/discount_requests/application)(`https://education.github.com/discount_requests/application`), the browser jumps to the `https://github.com/pricing` interface, what should I do?

**Comment by Kyushu21** [comment]
It sounds like there might be an issue with the link provided in the email. Here are a few steps you can take to address the problem:

Double-Check the Link: Make sure you're clicking on the correct link provided in the email. Sometimes, URLs can be truncated or contain extra characters, leading to redirection issues.
Contact GitHub Support: If you're still encountering problems after confirming the link, reach out to GitHub's support team. They can provide assistance and clarify any confusion regarding the verification process.
Verify Academic Status via Alternative Means: If you're unable to resolve the issue with the provided link, explore other options for verifying your academic status. This might include contacting your educational institution or exploring alternative verification methods offered by GitHub Education.
Check for Updates or Notices: Keep an eye out for any updates or notices from GitHub Education regarding this issue. They might be aware of the problem and working to resolve it, or they might provide alternative instructions for verification.

**Comment by gurjeetsinghvirdee** [comment]
Hi @imJack6 
Better, you can sign in here at `https://education.github.com/`, then click on your profile and select your benefit application. It will take you there, and you can easily re-upload your academic document
<img src="https://github.com/community/community/assets/73753957/4e49cf21-71da-49e0-8ff3-dc38ea96d299" width="500" />

**Reply by imJack6** [reply]
I tried it, still the same as before.

**Reply by gurjeetsinghvirdee** [reply]
then in that case mail them, and attach a video as a proof that **url** redirecting to other page

**Comment by peekcodedev** [comment]
If you are having trouble trying to verify your academic status on GitHub Education and are instead redirected to the GitHub pricing page, there are several steps you can try to resolve the issue:

1. **Check Your Email Again**:
    - Make sure you click the correct link in the email from GitHub Education. Often fake or incomplete links can lead you to the wrong page.

2. **Use Another Browser or Incognito Mode**:
    - Try using a different browser or incognito mode to access the link. Sometimes this problem is related to cookies or browser cache.

3. **Clear Cache and Cookies**:
    - Clear your browser's cache and cookies, then try accessing the link again. This can help fix problems related to temporary storage information on your browser.

4. **GitHub Support Contact**:
    - If you continue to experience problems, we recommend contacting GitHub support directly for assistance. You can send an email or visit their support page to get a solution.

5. **Check Your Academic Status**:
    - Make sure you have the necessary documents or information to verify your academic status. This can be a student identity card, a certificate from an educational institution, or another valid document.

6. **Fill in the Form Correctly**:
    - Make sure you fill out the verification form with correct and complete information. Errors or incomplete information can cause the verification process to fail.

7. **Wait for Confirmation**:
    - After submitting verification, wait some time to get confirmation from GitHub Education. The verification process can take several days, so be patient.

If after trying the steps above the problem is still not resolved, we recommend that you contact GitHub support directly for further assistance. They will be more experienced in dealing with problems like this and can provide the right solution.
